inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseExporterTest.java

```diff
@@ -105,8 +121,56 @@ public void thatExportingWorks() throws Exception
         verify(kbService, times(numOfLocalKBs)).importData(any(),
                 any(), any());
 
-        assertThat(exportedKbs).usingFieldByFieldElementComparator()
-                .containsExactlyInAnyOrderElementsOf(knowledgeBases());
+//        assertThat(exportedKbs).usingFieldByFieldElementComparator()
```

Comment:
Why are you weakening the tests here? We imho really need this check.

Reply:
Try putting in the old check and tell me how you'd do it. Thing is that on re-import under normal circumstances actually not all fields are the same, in particular the `project` and `repositoryId` usually mismatch... and they mismatch now that I made these changes to the test.

Reply:
@Rentier nevermind... found it :) Thanks for the prodding.

```
assertThat(exportedKbs)
                .usingElementComparatorIgnoringFields("repositoryId", "project")
                .containsExactlyInAnyOrderElementsOf(knowledgeBases());
```